src/applet.c

```diff
@@ -231,20 +229,18 @@ static void brisk_menu_applet_init(BriskMenuApplet *self)
         gtk_widget_set_margin_end(image, 4);
         gtk_widget_set_halign(image, GTK_ALIGN_START);
 
-        /* Set the label text, if empty, show the default text */
-        gchar *text = g_settings_get_string(self->settings, "label-text");
-        if (g_str_equal(text, "")) {
-                self->label_text = _("Menu");
-        } else {
-                self->label_text = text;
-        }
-
         /* Now add the label */
-        label = gtk_label_new(self->label_text);
+        label = gtk_label_new(_("Menu"));
         self->label = label;
         gtk_box_pack_start(GTK_BOX(layout), label, TRUE, TRUE, 0);
         gtk_widget_set_margin_end(label, 4);
 
+        /* Set the custom label text if the setting is not empty */
+        label_text = g_settings_get_string(self->settings, "label-text");
```

Comment:
This is still leaking. You must `g_free(label_text)` when you're done with it, the GtkLabel will `g_strdup` it's own copy.